src/sentry/static/sentry/app/views/onboarding/configure.jsx

```diff
@@ -23,19 +23,15 @@ class Configure extends React.Component {
 
   componentDidMount() {
     const {organization, params} = this.props;
-    const data = {
+
+    trackAnalyticsEvent({
```

Comment:
So do these now go to both reload and amplitude?

Reply:
They were already going to both. yes